src/odemis/acq/stream/_projection.py

```diff
@@ -41,6 +45,8 @@ def __init__(self, stream):
         stream (Stream): the Stream to project
         '''
         self.stream = stream
+        self.name = stream.name
+        self. acquisitionType = stream.acquisitionType
```

Comment:
This space after `self.` is very suspicious. Actually, I don't think you even need `.acquisitionType` anyway. Up to the caller to know that it's a projection, and look at `.stream.acquisitionType`?

Reply:
Ok. I think I added it for that reason - some view port tried to access the acquisitionType and broke.